packages/calypso-build/package.json

```
@@ -21,6 +21,7 @@
 	"bugs": {
 		"url": "https://github.com/Automattic/wp-calypso/issues"
 	},
+	"bin": "./bin/build-package.js",
```

Do you know how `npx` plays with packages with multiple bins? I don't off the top of my head.

It seems like we'll have multiple bins here eventually.

As some point we might want `cb` (Calypso build, props @blowery). I don't know if we'd want a single bin like `cb package` or `cb-package`?

We don't need to change it here, but worth thinking about.

```suggestion
	"bin": { "cb-package": "./bin/build-package.js" },
```

---

> It seems like we'll have multiple bins here eventually.

Wasn't so sure about that. I thought building a generic package build script might cut it? Was the reason why I was going with YAGNI anyway.

Anyway, `npx` can handle multiple bins, it's something like

```
npx --package @automattic/calypso-build cb-package
```

---

Want me to change it to that effect already? Might be less confusing for future contributors.

---

I'm not sure so no need to change here 🙂